ci/test/03_test_script.sh

```diff
@@ -162,8 +162,6 @@ if [ "${RUN_TIDY}" = "true" ]; then
     echo "^^^ ⚠️ Failure generated from clang-tidy"
     false
   fi
-  # Filter out files by regex here, because regex may not be
-  # accepted in src/.bear-tidy-config
```

Comment:
So is the filtering actually needed anymore, if it was only needed for `bear`? It'd be good to clarify if it is, or at the same time this comment is dropped, the redundant code should also be dropped?

Reply:
I'm not sure if I understand this answer, but, if the filtering is still needed, shouldn't the comment be updated instead to mention is is required for CMake?

Reply:
Ok, so I guess this comment should instead be changed to mention that it is still required by CMake.

Reply:
> Yes, the filtering is still needed. If the filtering is moved to the cmake side, I'd say a follow-up would be better.

I've just finished my tests. Apparently, filtering is no longer required. I'll submit a PR shortly and we can observe the IWYU output.

Reply:
> Ok, so I guess this comment should instead be changed to mention that it is still required by CMake. (Otherwise there is no context for why the filtering is happening or needed).

The comment has been adjusted.